Library/Formula/predictionio.rb

```diff
@@ -2,33 +2,38 @@
 
 class Predictionio < Formula
   homepage 'http://prediction.io/'
-  url 'http://download.prediction.io/PredictionIO-0.7.3.zip'
-  sha1 '5380f7f7d447ec04371753b060822e004c76a5fa'
+  url 'http://download.prediction.io/PredictionIO-0.8.5.tar.gz'
+  sha1 '1cc68733d2f8dbad1ae01e960f74085db24c7e74'
 
-  depends_on 'mongodb'
+  depends_on 'elasticsearch'
   depends_on 'hadoop'
+  depends_on 'hbase'
+  depends_on 'apache-spark'
+  depends_on :java => "1.7"
 
   def install
     rm_f Dir["bin/*.bat"]
 
     libexec.install Dir['*']
-    bin.write_exec_script Dir["#{libexec}/bin/*"]
+    bin.write_exec_script libexec/"bin/pio"
 
-    mv "#{bin}/start-all.sh", "#{bin}/predictionio-start-all.sh"
-    mv "#{bin}/stop-all.sh", "#{bin}/predictionio-stop-all.sh"
+    inreplace libexec/"conf/pio-env.sh" do |f|
+       f.gsub! /#\s*ES_CONF_DIR=.+$/, "ES_CONF_DIR=#{HOMEBREW_PREFIX}/opt/elasticsearch/config"
+       f.gsub! /SPARK_HOME=.+$/, "SPARK_HOME=#{HOMEBREW_PREFIX}/opt/apache-spark"
```

Comment:
use `s` as the block parameter name, and use `Formula["formula name"].opt_prefix` rather than hard coding `opt`